app/utilities/dhbDisplayName.ts

```diff
@@ -8,7 +10,7 @@ const nameMap: Record<string, string> = {
   "Hutt Valley": "Hutt Valley",
   Lakes: "Lakes",
   MidCentral: "MidCentral",
-  "Nelson Malborough": "Nelson Malborough",
+  "Nelson Marlborough": "Nelson Marlborough",
   "Overseas / Unknown": "Overseas / Unknown",
   Northland: "Northland",
```

Comment:
Is there any point in mapping names which don't have a translation?

Reply:
I reckon kill it